docs/UserGuide.md

```diff
@@ -65,129 +65,125 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
-Shows a list of all persons in the address book.
+### Deleting a person: `del`
 
-Format: `list`
+Delete a person from our contact list.
+
+Format: `del INDEX`
+
+Examples:
+* `del 1` deletes the first person in the contact list.
 
-### Editing a person : `edit`
 
-Edits an existing person in the address book.
+### Adding a tag : `tag-add`
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Add tags to a selected person from our contact list.
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+Format: `tag-add INDEX  TAG_NAME`
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `tag-add 1 Public Relations` adds the tag `Public Relations` to the first person in the contact list.
 
-### Locating persons by name: `find`
 
-Finds persons whose names contain any of the given keywords.
+### Deleting a tag : `tag-del`
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Add tags to a selected person from our contact list.
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+Format: `tag-del INDEX  TAG_INDEX`
 
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+* `tag-del 1 1` deletes the first tag from the first person in the contact list.
 
-### Deleting a person : `delete`
 
-Deletes the specified person from the address book.
+### Locating persons by keyword : `find`
 
-Format: `delete INDEX`
+Find persons matching any of the given keywords from our contact list.
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `find n/Alan t/team1`
```

Comment:
Do you think it's better to have 2 more examples to show that at least one field should be present?
I am suggesting adding
2) find n/Alan
3) find t/team1